homeassistant/components/plant/config_flow.py

```diff
@@ -0,0 +1,175 @@
+"""Config flow for Plant."""
```

Comment:
Please also note that 100% test coverage is required for config flows.

Reply:
I will look into that!